### ISSUE TITLE
github plugin

### ISSUE THREAD

**Issue by josephholsten** [driusan/de#42]
it would be awesome to have a github plugin with the same feature set as redmine.

**Comment by driusan** [comment]
You can look into making one with https://github.com/google/go-github.

But I've actually become convinced that the Redmine plugin would have been more useful as a redmine command-line app that prints to stdout in the format that the Redmine plugin modifies the buffer in, so it might be more beneficial to look into adding github support that way.

**Comment by josephholsten** [comment]
yeah, apparently I've volunteered to help maintain the github provider for terraform, so I'll be learning that package soon.
